types/index.d.ts

```diff
@@ -423,6 +423,10 @@ export interface ChannelListProps
     thisArg: React.Component<ChannelListProps>,
     e: Client.Event<Client.ChannelTruncatedEvent>,
   ): void;
+  onChannelHidden?(
+    thisArg: React.Component<ChannelListProps>,
+    e: Client.Event<Client.ChannelTruncatedEvent>,
```

Comment:
Change it to `Client.ChannelHiddenEvent`. It doesn't exist in js client yet. I will add it tomorrow :)

Reply:
Done in 38a08e6

Reply:
Thanks :)